api/src/main/java/org/pmiops/workbench/ras/RasLinkService.java

```diff
@@ -154,10 +152,9 @@ public DbUser linkRasAccount(String authCode, String redirectUrl) {
 
     String username = getUsername(userInfoResponse);
     DbUser user;
-    if (username.toLowerCase().contains(ID_ME_IDENTIFIER_LOWER_CASE)) {
-      user = userService.updateRasLinkIdMeStatus(username);
-    } else if (username.toLowerCase().contains(LOGIN_GOV_IDENTIFIER_LOWER_CASE)) {
-      user = userService.updateRasLinkLoginGovStatus(username);
+    if (username.toLowerCase().contains(ID_ME_IDENTIFIER_LOWER_CASE)
```

Comment:
Consolidating for now, but will probably end up splitting up again when I start track identity verifications in its own table.